profiles/tests/test_ordering.py

```diff
@@ -10,47 +10,200 @@
 )
 
 from profiles.factories import SavedCompanyFactory, SavedStartupFactory
+from utils.unittest_helper import utc_datetime
 from utils.dump_response import dump  # noqa
 
 
 class TestProfileOrdering(APITestCase):
     def setUp(self) -> None:
         self.user = UserFactory()
-        self.company_retail = ProfileCompanyFactory(name="Retail company")
-        self.company_winery = ProfileCompanyFactory(name="Winery")
-        self.company_delivery = ProfileCompanyFactory(name="Delivery company")
-        self.startup_catering = ProfileStartupFactory(name="Catering service")
-        self.startup_brewery = ProfileStartupFactory(name="Brewery")
-        self.startup_bakery = ProfileStartupFactory(name="Bakery")
 
-        self.saved_company_first = SavedCompanyFactory(user=self.user, company=self.company_delivery, added_at=(timezone.now() - timedelta(days=1)))
-        self.saved_company_second = SavedCompanyFactory(user=self.user, company=self.company_winery, added_at=timezone.now())
+        self.company_retail = ProfileCompanyFactory(
+            name="Retail company", completeness=105
+        )
+        self.company_retail.created_at = utc_datetime(2023, 12, 7)
+        self.company_retail.save()
 
-        self.saved_startup_first = SavedStartupFactory(user=self.user, company=self.startup_bakery, added_at=(timezone.now() - timedelta(days=1)))
-        self.saved_startup_second = SavedStartupFactory(user=self.user, company=self.startup_catering, added_at=timezone.now())
+        self.company_winery = ProfileCompanyFactory(
+            name="Winery", completeness=1
+        )
+        self.company_winery.created_at = utc_datetime(2023, 12, 5)
+        self.company_winery.save()
 
-    def test_get_list_of_companies_alphabetical_order(self):
-        response = self.client.get(path="/api/profiles/?is_registered=True&ordering=name&page=1&page_size=12")
-        companies_names = [company["name"] for company in response.data["results"]]
+        self.company_delivery = ProfileCompanyFactory(
+            name="Delivery company", completeness=2
+        )
+        self.company_delivery.created_at = utc_datetime(2024, 1, 15)
```

Comment:
Maybe, we need to update tests_new_members too

Reply:
I've updated new_members tests. 
![image](https://github.com/ita-social-projects/Forum/assets/114805677/6c61dde7-3f80-48f6-a8cf-8a84138993c6)

Reply:
I mean with utc_datetime util